i18n/locales/en.ts

```diff
@@ -1027,7 +1027,7 @@ const translations = {
       paragraph_two:
         'There are two 32-byte numbers we need to extract. They are referred to as R and S respectively and are each prefixed by the bytes 0220 in the DER sequence. Instead of fully decoding the DER blob, just look for the prefixes and paste the R and S values.',
       paragraph_three:
-        "# Satoshi's signature, from the input scriptSig of the tx to Hal Finney",
+        "# Satoshi's signature, from the input scriptSig of the tx to Hal Finney (block 170)",
```

Comment:
In general I think it's really helpful to refer to block 9 and block 170 whenever we can in this chapter because we're working with two transactions (the one from block 9 where Satoshi got the coins that were spent in block #170), and the output of the first transaction is the input of the second, meaning a specific UTXO can be considered both an output and an input and that's just all around confusing, even to the seasoned developer.

I plan to do another pass to identify more places where we can add clarification on if the text is referring to the transaction in block 9 or block 170